### PR TITLE
Update Gradle version

### PR DESCRIPTION
**Pull request by loay**
Gradle Version 2.10.0

Connect to #12

### CODE REVIEW

File: LoopbackGuideApplication/build.gradle [hunk 1]
```diff
@@ -3,19 +3,19 @@ buildscript {
...
-    compile 'com.google.android.gms:play-services:4.0.30'
+    compile 'com.google.android.gms:play-services:8.4.0'
     compile fileTree(dir: 'libs', include: '*.jar', exclude: 'android-support-v4.jar')
 }
 
 android {
-    compileSdkVersion 18
-    buildToolsVersion "19.0.0"
+    compileSdkVersion 23
+    buildToolsVersion "23.0.0"
```
Comment (bajtos) [inline LoopbackGuideApplication/build.gradle]: Please check with @rmg that the Mac build slave used for Android builds have this new version Android SDK and BuildTools installed.

Comment (bajtos) [inline LoopbackGuideApplication/build.gradle]: NVM, the build is passing and I did check that it actually compile the projects.

**Comment by bajtos** [comment]
Landed, thank you.

**Comment by bajtos** [comment]
@loay while you are at upgrading Gradle, could you please check loopback-sdk-android too? I think it's running an old version of Gradle too.

**Comment by loay** [comment]
@bajtos Will do.

**Pull request merged**